dev/breeze/src/airflow_breeze/utils.py

```diff
@@ -0,0 +1,51 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import hashlib
+import shlex
+import subprocess
+from typing import Dict, List
+
+from airflow_breeze.console import console
+
+
+def run_command(cmd: List[str], *, check: bool = True, verbose: bool = False, **kwargs):
+    if verbose:
+        console.print(f"$ {' '.join(shlex.quote(c) for c in cmd)}", style="info")
```

Comment:
I tihnk this one produces a little "too colorful" output :) 

![Screenshot from 2022-01-03 17-52-02](https://user-images.githubusercontent.com/595491/147957467-b9b3129f-329e-431d-bf0c-6236a4a949ef.png)

Reply:
I am currently going with color-coding in each log for now. Theme in python rich didn't work as expected when i pass that as parameter for console instance creation